Notes.md

```diff
@@ -0,0 +1,14 @@
+
+## Preface 
+A potential implementation of the AEG Data Structure.
+The AEG Tree will be broken up into individual nodes, with each node containing a list of atoms (signiyfing propositions) and a truth value indicating whether the node has a cut.
```

Comment:
Atoms need to be their own nodes as they can have arbitrary positions within a cut. Cuts should also be their own nodes. More comments on this in the Node file.